tests/cmd/auth/commands.go

```diff
@@ -64,6 +82,13 @@ func Login(user model.User) {
 	Eventually(sess).Should(Say(fmt.Sprintf("Logged in as %s\n", user.Username)))
 }
 
+// RegisterAndLogin registers a user using a randomized username and then logs in as the registered user.
+func RegisterAndLogin() model.User {
```

Comment:
@mboersma what do you think about the new use of `RegisterAndLogin`?  I'd also be fine with just keeping it all in `Register` _but_ the code does need to change from registering w/ `--login=false` and then logging in a user separately b/c:
```
// The 'login' flag is set to false during registration because otherwise the newly registered user's configuration
// will be automatically written to the file/profile used to register -- which in this case is the admin user's.
// Since we don't want to overwrite the admin user's configuration, we log the new user in separately.
```